day_7/fizzbuzz.rb

```diff
@@ -0,0 +1,40 @@
+x = 1
+
+while x <= 100
+  if x % 3 == 0
+    puts "Fizz"
+  elsif x % 5 == 0
+    puts "Buzz"
+  else
+    puts x
+  end
+  x += 1
+end
```

Comment:
@estein1988 - This looks to be a great start! One part you may want to revisit is for the case where `x` is both a multiple of 3 and a multiple of 5?  I think in the current state it's jumping into the first if statement and implicitly returning afterwards.